### DISCUSSION TITLE
Current installation guidelines don't work

### DISCUSSION THREAD

**Discussion by haleslj** [dipy/dipy#2544] [3. Community Lounge]
The current installation guidelines (Documentation 1.4.1) instructs the user to use anaconda to install dipy and fury. But this will install dipy version 1.4.1 and fury version 0.8.0 which aren't compatible.

**Comment by skoudoro** [comment]
Thank you for the feedback @haleslj , we are aware of this point and the future DIPY release (this week) should fix it
